docs/source/tensorbay_cli/dataset_management.rst

```diff
@@ -2,182 +2,199 @@
  Dataset Management
```

Comment:
actually we have two `dataset_management.rst` in the repo.
make your commit message more clear